espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/model/AttributesEnum.kt

```diff
@@ -106,6 +106,6 @@ abstract class CommonAttributes(var attributes: List<AttributesEnum>) {
     }
 
     fun valueOf(value: String): AttributesEnum? {
-        return attributes.find { it.toString().equals(value, ignoreCase = false) }
+        return attributes.find { it.toString().equals(value, ignoreCase = true) }
```

Comment:
what is the reason to do this change?

Reply:
Checking the case didn't make sense to me. I can revert back if needed, but not sure why we need to respect case in this case.
e.g.: "CHECKED" == "checked"

Reply:
because attribute names in xml are in lowercase. Also the change itself has nothing to with this PR

Reply:
Could see your point, reverted.